src/index.html

```diff
@@ -5,9 +5,39 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+      <h3 class="card__title" >APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+      <p class="card__product-code" >Product code: 195434</p>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+      <p class="card__price">
+        <span class="card__price-name">Price:</span>
+        <span class="card__price-number">$2,199</span>
+      </p>
+      <button class="card__button" data-qa="hover">
+        buy
+      </button>
```

Comment:
Consider will be better if you add the empty line between the elements
```suggestion
      <img
        class="card__image"
        src="images/imac.jpeg"
        alt="imac"
      >
      
      <h3 class="card__title" >APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
      
      <p class="card__product-code" >Product code: 195434</p>
      
      <div class="card__rating">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        
        <p class="card__reviews">Reviews: 5</p>
      </div>
      
      <p class="card__price">
        <span class="card__price-name">Price:</span>
        
        <span class="card__price-number">$2,199</span>
      </p>
      
      <button class="card__button" data-qa="hover">
        buy
      </button>
```